### ISSUE TITLE
Rule Proposal: Analog of vue/match-component-file-name for imported components

### ISSUE THREAD

**Issue by ilyub** [vuejs/eslint-plugin-vue#1786]
**Please describe what the rule should do:**

This rule should enforce consistent tag names for imported components.

**What category should the rule belong to?**

Enforces code style (layout)

**Provide 2-3 code examples that this rule should warn about:**

Valid example
```vue
import IconPicker from "./IconPicker.vue";

export default defineComponent({
  name: "app",
  components: {
    "icon-picker": IconPicker
```

Invalid example
```vue
import IconPicker from "./IconPicker.vue";

export default defineComponent({
  name: "app",
  components: {
    "some-inconsistent-name": IconPicker
```

**Comment by FloEdelmann** [comment]
Good idea! I'd suggest `vue/match-component-import-name` as a rule name.

**Comment by ilyub** [comment]
Thx for the update
